package.json

```diff
@@ -18,30 +18,39 @@
 	"dependencies": {
 		"@hookform/resolvers": "^3.6.0",
 		"@reduxjs/toolkit": "^2.2.5",
+		"@testing-library/user-event": "^14.5.2",
```

Comment:
let's also this be installed as dev dependency: "@testing-library/user-event": "^14.5.2",